vignettes/tutorial2.Rmd

```diff
@@ -0,0 +1,447 @@
+---
+title: "Fldgen v2 Tutorial - Joint Temperature and Precipitation emulation and function how to"
+author: "Abigail Snyder"
+date: "`r Sys.Date()`"
```

Comment:
This will get updated every time the Vignette is regenerated.  I'd put a real date here reflecting when the text was actually written.